packages/api/src/routes/medical/internal-patient.ts

```diff
@@ -486,4 +492,29 @@ router.get(
   })
 );
 
+router.post(
+  "/trigger-update",
```

Comment:
It's odd that we have two endpoints to update all patients:
- `POST /internal/patient/update-all`
- `POST /internal/patient/trigger-update`

Unfortunately the `update-all` was not named specifically to CW.

I'd suggest we either:
- treat this new one as temporary, and remove it once it's work is done; or
- rename it (and maybe the other one) so they're easier to understand/maintain.

Also, this is missing a doc, we've been doing this for all endpoints for some time now.